docs/metrics/nli-classification.md

```diff
@@ -0,0 +1,57 @@
+# NLI Classification
```

Comment:
Should we name it by the metric's name?
Are we going to have this doc focused on `contradiction` class so we can identify hallucination?